nx-X11/programs/Xserver/render/render.c

```diff
@@ -2918,9 +2920,8 @@ PanoramiXRenderFillRectangles (ClientPtr client)
 	    result = (*PanoramiXSaveRenderVector[X_RenderFillRectangles]) (client);
 	    if(result != Success) break;
 	}
-	free(extra);
     }
-
+    free(extra);
```

Comment:
moving that one level up might free an unitialized pointer. On first glance it looks like the original location of the free was correct.

Reply:
removed